doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -108,51 +160,100 @@ As a _superuser_ (`cloudsqlsuperuser`):
 
 {{< /tabs >}}
 
-Once logical replication is enabled:
+## Step 2. Create a publication
 
-1. Grant the required privileges to the replication user:
+Once logical replication is enabled, the next step is to create a publication with the tables that you want to replicate to Materialize. You'll also need create a user for Materialize and give the user permissions to manage replication.
+
+{{< tabs >}}
+{{< tab "AWS RDS">}}
+
+1. For each table that you want to replicate to Materialize, set the [replica identity](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY) to `FULL`:
 
     ```sql
-    ALTER ROLE "dbuser" WITH REPLICATION;
+    ALTER TABLE table1 REPLICA IDENTITY FULL;
     ```
 
-    And ensure that this user has the right permissions on the tables you want to replicate:
+    This setting makes sure sufficient information is written to the replication stream for  `UPDATE` and `DELETE` operations.
+
+    <!-- THE LINKED PAGE DOESN'T TALK ABOUT INCREASED CPU. WE SHOULD FIND ANOTHER PAGE TO LINK TO. COMMENTING THIS OUT IN THE MEANTIME. As a heads-up, you should expect a performance hit in the database from increased CPU usage. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/logical-replication-publication.html). -->
```

Comment:
I think the conversation about increased CPU usage is a little vague--what is more pressing is the increased disk usage (you have to store more data in the WAL).  I think quantifying it this way also alleviates some user apprehension because disk space is cheap. I don't know that a larger WAL _really_ demands more out of the CPU. It's like saying a hard drive that has less free space offers less CPU.